extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/sql/resources/SqlStatementResourceTest.java

```diff
@@ -916,17 +917,17 @@ public void testMSQInsertRunningQuery()
   }
 
   @Test
-  public void testForbiddenRequest()
+  public void testAPIBehaviourWithSuperUsers()
   {
     Assert.assertEquals(
-        Response.Status.FORBIDDEN.getStatusCode(),
+        Response.Status.OK.getStatusCode(),
         resource.doGetStatus(
             RUNNING_SELECT_MSQ_QUERY,
             makeExpectedReq(CalciteTests.SUPER_USER_AUTH_RESULT)
         ).getStatus()
     );
     Assert.assertEquals(
-        Response.Status.FORBIDDEN.getStatusCode(),
+        Response.Status.BAD_REQUEST.getStatusCode(),
```

Comment:
Why has this changed from forbidden to bad request ?

Reply:
We can access the status of the query therefore it is no longer forbidden.
Since it is a running query, requesting the results returns in the BAD_REQUEST response code (as is the case if the user submitting the query requests the results before completion)